setup.py

```diff
@@ -55,7 +55,6 @@ def find_version(*file_paths):
         'flask>=0.10.0',
         'lxml>=3.4.4',
         'spyne>=2.12.8',
-        'nose',
```

Comment:
We can re-add `odm2api` as a dev dependency. This will allow for vanilla WOFpy installations while advanced users will know what to do.